nixos/modules/services/web-apps/hatsu.nix

```diff
@@ -0,0 +1,101 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+let
+  cfg = config.services.hatsu;
+in
+{
+  meta.doc = ./hatsu.md;
+  meta.maintainers = with lib.maintainers; [ kwaa ];
+
+  options.services.hatsu = {
+    enable = lib.mkEnableOption "Self-hosted and fully-automated ActivityPub bridge for static sites";
+
+    package = lib.mkPackageOption pkgs "hatsu" { };
+
+    dataDir = lib.mkOption {
+      type = lib.types.str;
+      default = "/var/lib/hatsu";
+      description = "Data directory for hatsu.";
+    };
```

Comment:
I don't think a custom datadir (not managed by systemd) would work well with DynamicUser. We might drop this option as well.

Reply:
Fixed as `/var/lib/hatsu`